client/src/styles/PropertiesMap.scss

```diff
@@ -40,35 +40,62 @@
       }
     }
 
-    .warning-banner {
+    .MapBanner__container {
       position: absolute;
       z-index: 100;
       top: 0;
-      margin: 1rem 1.1rem 0 4.5rem;
-      background-color: $wowza-orange;
-      color: $wowza-dark;
-      border-radius: 0.2rem;
-      padding: 0.5rem 2.1rem 0.5rem 1.08rem;
-      min-height: 5.4rem;
+      left: 0;
+      right: 0;
+      margin: 1rem 2.2rem 0 4.5rem;
       display: flex;
-      font-size: 1.4rem;
+      flex-direction: column;
 
-      .float-left {
+      .MapBanner {
         display: flex;
-        align-items: center;
-        padding: 0;
+        position: relative;
+        min-height: 5.4rem;
+        padding: 0.5rem 1.7rem 0.5rem 4.2rem;
+        margin-bottom: 0.5rem;
+        border-radius: 0.2rem;
+        font-family: "Inconsolata", monospace;
```

Comment:
small: might be nice to eventually move this font to a sass variable as well

Reply:
oh yeah for sure good idea, I'll check with Tahnee about how this fits into the design system and do this later.